### ISSUE TITLE
Can this tool automatically decode in the Proxy tab？

### ISSUE THREAD

**Issue by f0ng** [hackvertor/hackvertor#102]
Perfect tool！
By marking '<@ d' in the response_ Base64>dGVzdA==</@ d_ Base64>'Display back and forth as' test'. Currently, decoding can only be done through additional tabs. I want to achieve automatic decoding,Can this tool meet my needs?

**Comment by hackvertor** [comment]
Yep you can use Hackvertor in the proxy but it is disabled by default. You can change this in the Hackvetor menu in the main menu bar. Note the reason this is off by default is security risks, meaning a site could inject their own Hackvetor tags. If you are ok with this risk and you trust the site(s) in question then it should be ok to enable this option.

**Comment by f0ng** [comment]
I enabled this option, but instead of automatically decoding it, I decoded it in the extension tab

![image](https://github.com/hackvertor/hackvertor/assets/48286013/0332a6c6-64a6-4cbc-a827-0c9ad3071485)

<img width="1159" alt="image" src="https://github.com/hackvertor/hackvertor/assets/48286013/bc53b8a7-020d-4079-a420-283e4b349997">

What should I do to automatically decode in response instead of manually decoding in the extension tab

**Comment by hackvertor** [comment]
There's no current way of decoding tags in the response. I'm thinking of a way to do this as it was reported in other issues. Sorry about that cheers

**Comment by AkikoOrenji** [comment]
Along these same lines is it possible to set a set of default tags to cover the whole response in the proxy HTTP history ? I'm guessing not based on the above but wanted to confirm.   I have a server that responds with a unicode escaped JSON data for every request. Would be nice if i could use Hackvertor to automatically decode the unicode escape sequences in the response viewer and then burp could just pretty print the json. 
Thanks

**Comment by f0ng** [comment]
Pretty good



发自我的iPhone


------------------ Original ------------------
From: Gareth Heyes ***@***.***&gt;
Date: Fri,Sep 12,2025 8:09 PM
To: hackvertor/hackvertor ***@***.***&gt;
Cc: f0ng ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [hackvertor/hackvertor] Can this tool automatically decode in the Proxy tab？ (Issue #102)



hackvertor left a comment (hackvertor/hackvertor#102)
 
I've created a new feature called Profiles. This allows you to run some Python to detect the encoding and modify it and add tags. This should do the job for you. It's in Hackvertor v2.1.7.
